build/int.cloudbuild.yaml

```diff
@@ -162,12 +162,20 @@ steps:
       - appinfra-apply
       - fleetscope-verify
 
-  - id: app-e2e
+  - id: cymbal-bank-e2e
     name: "gcr.io/cloud-foundation-cicd/$_DOCKER_IMAGE_DEVELOPER_TOOLS:$_DOCKER_TAG_VERSION_DEVELOPER_TOOLS"
     args:
       ["/bin/bash", "-c", "cft test run TestAppE2E --stage verify --verbose"]
```

Comment:
It would be nice to rename the test from `TestAppE2E` > `TestCymbalBankE2E` for clarity/consistency